sunpy/tests/setup_command.py

```diff
@@ -13,6 +13,7 @@
 
 from astropy_helpers.commands.test import AstropyTest
 from astropy_helpers.compat import _fix_user_options
+import importlib
```

Comment:
as `importlib` is part of python core should be added on the block above, not in here (line12).

Reply:
@dpshelio Fixed it!